content/v2.0/reference/cli/influx/pkg/_index.md

```diff
@@ -21,22 +21,25 @@ influx pkg [command]
-| `-h`, `--help`            | Help for the `pkg` command                                                            |            |
-| `-o`, `--org`             | The name of the organization that owns the bucket                                     | string     |
-| `--org-id`                | The ID of the organization that owns the bucket                                       | string     |
-| `-q`, `--quiet`           | Disable output printing                                                               |            |
-| `--secret`                | Secrets to provide alongside the package (format: `--secret=SECRET_KEY=SECRET_VALUE`) | string     |
-| `--disable-table-borders` | Disable table borders                                                                 |            |
+| Flag                      | Description                                                                                     | Input Type |
+|:----                      |:-----------------------------                                                                   |:---------- |
+| `-c`, `--disable-color`   | Disable color in output                                                                         |            |
+| `--disable-table-borders` | Disable table borders                                                                           |            |
+| `-e`, `--encoding`        | Encoding of the input stream                                                                    | string     |
+| `--env-ref`               | Environment references to provide alongside the package (format: `--env-ref=REF_KEY=REF_VALUE`) | string     |
+| `-f`, `--file`            | Path to package file                                                                            | string     |
+| `--force`                 | Ignore warnings about destructive changes                                                       |            |
+| `-h`, `--help`            | Help for the `pkg` command                                                                      |            |
+| `-o`, `--org`             | The name of the organization that owns the bucket                                               | string     |
+| `--org-id`                | The ID of the organization that owns the bucket                                                 | string     |
+| `-q`, `--quiet`           | Disable output printing                                                                         |            |
+| `-R`, `--recurse`         | Recurse through files in the directory specified in `-f`, `--file`                              |            |
+| `--secret`                | Secrets to provide alongside the package (format: `--secret=SECRET_KEY=SECRET_VALUE`)           | string     |
+| `-u`, `--url`             | URL of a package file file                                                                           | string     |
```

Comment:
package file file?

Reply:
Ha, thanks. I had copied this in three places 🤦‍♂